### ISSUE TITLE
Add responsive separator behavior for the canvas/zoom controls panel

### ISSUE THREAD

**Issue by cbeer** [ProjectMirador/mirador#2167]
### Responsive behavior
- [ ] If the canvas is narrower than can fit the entire row of zoom/nav buttons, they should float to two lines:

<img width="381" alt="responsive" src="https://user-images.githubusercontent.com/101482/54046347-437f7280-4191-11e9-8d7a-37a92ec2d229.png">

- [ ] Note that when they go to two lines, we should not display the vertical divider between the two sets of buttons.

**Comment by ggeisler** [comment]
I have two potential suggestions for responsiveness of the canvas nav controls and canvas title. One is a preferred approach but depends on us being able to know the current width of the canvas at all times (getting the updated value when the window is resized or the sidebar is opened, for example). Is it possible for us to know the canvas at all times?

If not, I'll post a less ideal approach that doesn't rely on knowing the canvas width.

**Comment by aeschylus** [comment]
Yes, we can always know the canvas width.

**Comment by ggeisler** [comment]
Okay, based on @aeschylus confirmation that we can always know the canvas width, here's the preferred solution:

1. Update the divider

Currently we're using a CSS content property to add a vertical divider character. Unfortunately, both @jvine and I feel this character doesn't work well in this context; it looks too chunky, and I don't think there is a way to style it to make it look better. The mockups below suggest another approach, which is to add a new, empty `<span />` element after the existing `<span title="Reset zoom" />` element, and give that new span these properties:

```css
span.vertical-divider {
  border-right: 1px solid #808080;
  display: inline-block;
  height: 24px;
  margin: 12px 6px;
}
```

2. When canvas width is at least 250px, there is enough room in a single row for the three zoom control buttons, the new vertical-divider `<span />`, and the two canvas nav buttons. So they are all displayed like this:

<img width="513" alt="controls-md" src="https://user-images.githubusercontent.com/101482/54467316-861cee00-4741-11e9-9d9f-9432b5355943.png">

3. When the canvas width is less than 250px, we display the zoom control buttons and the canvas nav buttons in two stacked and centered rows, and hide the new  vertical-divider `<span />`:

<img width="513" alt="controls-sm" src="https://user-images.githubusercontent.com/101482/54467375-f0359300-4741-11e9-9666-2cf27f944134.png">

4. If the zoom controls are off, the vertical-divider span is also hidden, of course. And if the zoom controls are turned on, we probably need to check the current window width to decide whether to show it (width is 250px or greater) or not (width is less than 250px).

**Comment by jvine** [comment]
I'll note that there's at least one other context coming up where this vertical divider will be useful.

**Comment by ggeisler** [comment]
Reopening this because the part about 

> Note that when they go to two lines, we should not display the vertical divider between the two sets of buttons

didn't seem to be included in the merged PR #2425. In the example below, the divider should be removed and the middle zoom button should be move a bit to the right so it is centered over the middle of the prev and next buttons.

<img width="246" alt="Screen Shot 2019-04-10 at 8 14 54 AM" src="https://user-images.githubusercontent.com/101482/55891473-78546000-5b69-11e9-89af-c9ecdad2234a.png">

I thought I checked for this when @jkeck asked me to review the PR yesterday but apparently I missed it.
